### ISSUE TITLE
[RFE] #34 report the lack of wheel chair accessibility

### ISSUE THREAD

**Issue by sebacodoni76** [grafuls/fiscalia#30]
Is your feature request related to a problem? Please describe.
that can issue an alert for differences or ploblems in lack of wheel chair accessebility 

Describe the Possible Solution
acceso dropdown menu of faults or problems to report
